### ISSUE TITLE
boosted random forest

### ISSUE THREAD

**Issue by salmontuna1** [microsoft/LightGBM#6294]
Is it possible to add boosted random forest feature? Similar to the xgboost feature num_parallel_tree. This will speed up the training time for the problems with a small number of features (in which using higher values of num_thread not making things faster because of overhead). If we can train couple of trees at the same time and use their averages for boosting that will make it possible to achieve similar accuracy in shorter time for the problems with small number of features.  One more suggestion for the implementation, I think num_thread should only used for generating one tree just as now and the parallel trees should create its own parallel threads with a different parameter (or num_parallel_tree can used for this purpose as well) , otherwise the overhead problem emerges again, which is the case in current xgboost implementation.

Thanks,

**Comment by AhmedThahir** [comment]
Any updates on this?

**Comment by jameslamb** [comment]
Thanks for your interest in LightGBM.

We'd welcome comments affecting the progress on this issue (e.g. offers to help contribute, information that changes the priority, references to competing or complementary implementations),  but please .... don't leave "any updates on this?" types of comments in this repo. You can click "subscribe" to be notified when there is new activity on a GitHub issue.

**Comment by AhmedThahir** [comment]
Hiiii, I'm already subscribed to this issue.
There was no comment on the roadmap so I wanted to follow up out of curiosity.

I understand your perspective and I'll keep in mind going forward. Thank you for pointing it out.
